NWEEI/NWEEI/Views/Resources/IndustryNews.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@model List<Article>
+@using System.Text.RegularExpressions;
+
+@{
+    ViewData["Title"] = "Industry News";
+}
+
+<h1>Industry News</h1>
+
+<div class="articles container">
+    @foreach (var article in Model)
+    {
+        <a controller="Article" asp-action="Details" asp-route-id="@article.ArticleID">
```

Comment:
I added a controller tag helper so that clicking an industry news article doesn't lead to a "page not found" error.